client/me/purchases/concierge-banner/index.jsx

```diff
@@ -32,6 +32,7 @@ class ConciergeBanner extends Component {
 			CONCIERGE_HAS_AVAILABLE_INCLUDED_SESSION,
 			CONCIERGE_HAS_AVAILABLE_PURCHASED_SESSION,
 			CONCIERGE_SUGGEST_PURCHASE_CONCIERGE,
+			'placeholder',
```

Comment:
It looks to me that showing a placeholder or not has been replaced by the `showPlaceholder` prop. Do we still need this?

Reply:
Yup, this change has been outdated. I've removed it.